broker-daemon/models/market-event.js

```diff
@@ -8,12 +8,13 @@ class MarketEvent {
     this.orderId = orderId
     this.timestamp = timestamp
     this.eventType = eventType
+    this.eventOrder = eventOrder
```

Comment:
I don't think we should use eventOrder as the name for this. It is confusing just because we use order and eventOrder in other places. Maybe ordering, sequence etc.

Reply:
@martineehrlich Going to rename this value to `sequence` instead